demo/src/app/components/popover/demos/autoclose/popover-autoclose.html

```diff
@@ -0,0 +1,85 @@
+<p>
+  As for some other popup-based widgets, you can set the popover to auto close upon some events.
+</p>
+
+<p>
+  The following one will close on clicks outside the popover, and on <code>Escape</code> presses.
+</p>
+
+<button
+  type="button"
+  class="btn btn-outline-secondary"
+  popoverTitle="Pop title"
+
+  [autoClose]="'outside'"
+  ngbPopover="Click outside or press Escape to close"
+
+  triggers="manual"
+  #popoverManualOpening1="ngbPopover"
+  (mouseenter)="popoverManualOpening1.open()"
+>
+    Hover to open
+</button>
+
+<p>
+  The following one will close on clicks inside the popover, and on <code>Escape</code> presses.
+</p>
+
+<button
+  type="button"
+  class="btn btn-outline-secondary"
+  popoverTitle="Pop title"
+
+  [autoClose]="'inside'"
+  ngbPopover="Click inside or press Escape to close"
+
+  triggers="manual"
+  #popoverManualOpening2="ngbPopover"
+  (mouseenter)="popoverManualOpening2.open()"
+>
+    Hover to open
+</button>
+
+<p>
+  The following one will close on clicks from anywhere, except its implicit toggling element (defined by <code>triggers</code>), and on <code>Escape</code> presses.
```

Comment:
I don't understand this and it actually closes when you click on the toggling element

Reply:
It was when the triggers were parsed to automatically exclude the toggling element. Now they're not anymore, so no need to tell about it. I should merge this example with the one having an external toggler.